src/components/dropdown/dropdown.stories.svelte

```diff
@@ -207,7 +207,7 @@
   .country {
     display: flex;
     flex-direction: row;
-    gap: 8px;
+    gap: var(--leo-spacing-xl);
```

Comment:
**Note:** This only affects the story, not the component - probably good for demonstration purposes :smile: